internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -45,13 +45,26 @@ username: %s
 password: %s
 `
 
+const quickstartTemplateIntro = `You are creating a new Atlas cluster and enabling access to it.
+
+Press [Enter] to use the default values.
+
+Enter [?] on any option to get help.
```

Comment:
[q] How does that work? If I type ? instead of enter I get a params list or can specify the parameters instead of the default values?

Reply:
If you press `?` you get a small description of what you should provide

```
Press [Enter] to use the default values.

Enter [?] on any option to get help.

[Set up your Atlas cluster]
? Name of the cluster.
? Cluster Name [This can't be changed later] (Quickstart-1625578411)
```

Reply:
Got it, thanks that makes sense